modules/gapi/perf/common/gapi_imgproc_perf_tests_inl.hpp

```diff
@@ -622,6 +622,54 @@ PERF_TEST_P_(CannyPerfTest, TestPerformance)
 
 //------------------------------------------------------------------------------
 
+PERF_TEST_P_(GoodFeaturesPerfTest, TestPerformance)
+{
+    compare_f cmpF;
+    MatType type;
+    int maxCorn = 0, blockSize = 0;
+    double qual = 0.0, minDist = 0.0, k = 0.0;
+    cv::Size sz;
+    bool useHarris = false;
+    cv::GCompileArgs compile_args;
+    std::tie(cmpF, type, sz, maxCorn, qual, minDist, blockSize, useHarris, k, compile_args) = GetParam();
+
+    initMatrixRandN(type, sz, CV_8UC1, false);
```

Comment:
Not sure if this is a correct way to test kernels like GoodFeatures.

Please find an appropriate image from `opencv_extra` and load it here. Baboon should work ok for the purpose:

![image](https://user-images.githubusercontent.com/144187/75979015-bb8d8f00-5ef0-11ea-8303-05d83866321c.png)

Reply:
done. Had to use initTestDataPath() function from Streaming test